src/abilities/Nutcase.js

```diff
@@ -206,10 +206,10 @@ export default G => {
 					target, // Target
 					'onStepOut', // Trigger
 					{
-						effectFn: function(effect) {
-							effect.target.takeDamage(
```

Comment:
Why are we going to less clear var names here?

Reply:
because it shadowed the same name outside